components/datatypes/b2b-account.schema.json

```diff
@@ -95,13 +95,6 @@
           "description": "Source type for the account.",
           "type": "string"
         },
-        "xdm:accountCurrencyCode": {
```

Comment:
I think we should not remove this property as it could be the default currency of the account and not really related to any amount properties. @peichiang , What is your thoughts about this ?